README.md

```diff
@@ -919,13 +919,42 @@ extension your_class : RideRequestViewControllerDelegate {
 
 ## Example Apps
 
-Example apps can be found in the `examples` folder. To run build them, you need to use Carthage. (A quick overview of installing Carthage can be found in the **Getting Started** section.) From inside the `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
+Example apps can be found in the `examples` folder. To run build them, you can use Carthage or Cocoapods. 
+
+### Carthage
+(A quick overview of installing Carthage can be found in the **Getting Started** section.) From inside the `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
 
 ```
 carthage update --platform iOS
 ``` 
 This will build the required dependencies. Once you do that, open `Swift SDK.xcodeproj` or `Obj-C SDK.xcodeproj` in Xcode and run it.
 
+### CocoaPods
+You will have to remove Carthage dependencies. Remove Cartfile and Cartfile.resolved.
+
+From `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
+To integrate Uber Rides into your Xcode project, navigate to the directory `examples/Swift SDK` or `examples/Obj-C SDK` and create a new **Podfile** with `pod init`, then add `pod 'UberRides'` to the main loop. If you are using the Swift SDK, make sure to add the line `use_frameworks!`.
```

Comment:
Looks like some copy & paste issues here, you reference the `examples/Swift SDK` or `examples/Obj-C SDK` twice